test/cli/fixtures/expected/tap-outputs.js

```diff
@@ -124,5 +124,39 @@ not ok 2 global failure
 # skip 0
 # todo 0
 # fail 2
+`,
+
+	"qunit no-tests":
+`TAP version 13
+not ok 1 global failure
+  ---
+  message: "No tests were run."
+  severity: failed
+  actual: null
+  expected: undefined
+  stack: undefined:undefined
```

Comment:
The `undefined:undefined` doesn't look too good. Can we avoid that in in a fairly easy way, or something for later?

Reply:
I agree but would prefer to address in a follow-up PR. It has to do with the current `QUnit.onError` implementation here https://github.com/qunitjs/qunit/blob/a577183e8d4f975d822e3021cbba552c5ed77549/src/core/onerror.js#L18